apps/custom-ui-floating-window-demo/components/ContactList.js

```diff
@@ -0,0 +1,61 @@
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+import { startOutgoingCall } from "../shared/socket";
+import Footer from "./Footer";
```

Comment:
Can you use single quotes in imports and string declarations? It seems to be more common standard these days and would match the pattern we use in other example apps

You could enable eslint and let linter fix them all for you, so it doesn't have to be done manually